appveyor.yml

```diff
@@ -27,30 +29,50 @@ environment:
   matrix:
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: ""
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
```

Comment:
I'm fine with env variable, but maybe this can be detected without it? (can be done later of course)

Reply:
Never mind, since without env var matrix.exclude will not work